mono/eglib/gmodule-win32.c

```diff
@@ -137,6 +137,16 @@ g_module_symbol (GModule *module, const gchar *symbol_name, gpointer *symbol)
 	}
 }
 
+gboolean
+g_module_address (void *addr, char **file_name, void **file_base, char **sym_name, void **sym_addr)
+{
+	/*
+	 * XXX: For an actual implementation, file_name can be filled in from
+	 * GetModuleFileName, at least?
+	 */
```

Comment:
`GetModuleHandleEx(GET_MODULE_HANDLE_EX_FLAG_FROM_ADDRESS)`.

Reply:
And then `GetModuleFileName` agreed.

Reply:
I don't have a Windows system anymore to test this on, so I would be flying blind. I did do some research into what a Windows implementation would likely use though, so I did leave the comment here.